src/components/molecules/map/GoogleMapsLayer.tsx

```diff
@@ -1,12 +1,21 @@
 import React, { FC } from 'react';
 import ReactLeafletGoogleLayer from 'react-leaflet-google-layer';
+import { useParams } from 'react-router-dom';
 import { MAP_STYLE } from 'style/map.style';
 import { mapApiKey } from 'utils/utils';
 
+interface lng {
+  lng: string;
+}
+
 const GoogleMapsLayer: FC = ({ children }) => {
+  const { lng }: lng = useParams();
```

Comment:
```suggestion
  const { lng } = useParams<lParam>();
```

Reply:
see https://stackoverflow.com/questions/64517983/useparams-in-typescript-does-not-allow-destructuring